x/concentrated-liquidity/types/msgs.go

```diff
@@ -87,12 +87,17 @@ func (msg MsgAddToPosition) ValidateBasic() error {
 		return fmt.Errorf("Invalid position id (%s)", strconv.FormatUint(msg.PositionId, 10))
 	}
 
-	if !msg.TokenDesired0.IsValid() || !msg.TokenDesired0.IsPositive() {
-		return fmt.Errorf("Invalid coins (%s)", msg.TokenDesired0.String())
+	if msg.Amount0.IsNegative() {
```

Comment:
Is there a reason you removed the validity check here too?

Reply:
Agreed on keeping validity check

Reply:
The reason validity check was removed was because we have changed this field from coins to int, thus there is nothing to call validity check anymore, lmk if this makes sense